src/domains/wallet-common/03_storage.tf

```diff
@@ -0,0 +1,228 @@
+resource "azurerm_resource_group" "storage_wallet_rg" {
+  name     = "${local.project}-storage-rg"
+  location = var.location
+  tags     = var.tags
+}
+
+
+module "wallet_storage_snet" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//subnet?ref=v6.7.0"
+
+  name                 = "${local.project}-storage-snet"
+  address_prefixes     = var.cidr_subnet_storage_wallet
+  resource_group_name  = local.vnet_resource_group_name
+  virtual_network_name = local.vnet_name
+
+  private_endpoint_network_policies_enabled = true
+
+  service_endpoints = [
+    "Microsoft.Storage",
+  ]
+}
+
+
+resource "azurerm_private_endpoint" "storage_private_endpoint" {
+  count = var.env_short != "d" ? 1 : 0
+
+  name                = "${local.project}-tr-storage-private-endpoint"
+  location            = var.location
+  resource_group_name = azurerm_resource_group.storage_wallet_rg.name
+  subnet_id           = module.wallet_storage_snet.id
+  private_dns_zone_group {
+    name                 = "${local.project}-storage-private-dns-zone-group"
+    private_dns_zone_ids = [data.azurerm_private_dns_zone.storage.id]
+  }
+
+  private_service_connection {
+    name                           = "${local.project}-storage-private-service-connection"
+    private_connection_resource_id = module.wallet_storage.id
+    is_manual_connection           = false
+    subresource_names              = ["queue"]
+  }
+
+  tags = var.tags
+}
+
+module "wallet_storage" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//storage_account?ref=v6.7.0"
+
+
+  name                            = replace("${local.project}-sa", "-", "")
+  account_kind                    = var.wallet_storage_params.kind
+  account_tier                    = var.wallet_storage_params.tier
+  account_replication_type        = var.wallet_storage_params.account_replication_type
+  access_tier                     = "Hot"
+  blob_versioning_enabled         = true
+  resource_group_name             = azurerm_resource_group.storage_wallet_rg.name
+  location                        = var.location
+  advanced_threat_protection      = var.wallet_storage_params.advanced_threat_protection
+  allow_nested_items_to_be_public = false
+  public_network_access_enabled   = var.wallet_storage_params.public_network_access_enabled
+
+  blob_delete_retention_days = var.wallet_storage_params.retention_days
+
+  network_rules = var.env_short != "d" ? {
+    default_action             = "Deny"
+    ip_rules                   = []
+    virtual_network_subnet_ids = [module.wallet_storage_snet.id]
+    bypass                     = ["AzureServices"]
+  } : null
+  tags = var.tags
+}
+
+resource "azurerm_storage_queue" "wallet_usage_update_queue" {
+  name                 = "${local.project}-wallet-usage-update-queue"
```

Comment:
`wallet` is already present in `local.project` (`pagopa-d/u/p-weu-wallet`), so you might think about removing it.
looking terraform plan the name of this resource would become `pagopa-d/u/p-weu-wallet-wallet-usage-update-queue`

Reply:
Fixed in 444830f2